projects/igniteui-angular/src/lib/paginator/paginator.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="igx-grid-paginator__select" [hidden]="dropdownHidden">
+    <label class="igx-grid-paginator__label">{{ selectLabel }}</label>
+    <div class="igx-grid-paginator__select-input">
+        <igx-select [(ngModel)]="perPage" [displayDensity]="paginatorSelectDisplayDensity()" type="border"
```

Comment:
With two-way binding this would be just `[(ngModel)]="perPage"` and no `(ngModelChange)` handler.